lesson06/assignment/css/style.scss

```diff
@@ -4,3 +4,239 @@
 @import 'settings.responsive';
 @import 'components.content';
 
+// Import partials here
+@import 'base.normalize';
+@import 'settings.variables';
+@import 'settings.responsive';
+@import 'components.content';
+
+@keyframes header {
+    from {
+        background-color: #138DD3
+    }
+
+    to {
+        background-color: #189E20;
+    }
+}
+
+//declaring variables
+$text-color-1: white;
+$font: "arial, helvetica, sans-serif";
+
+//using mixins
+@mixin header-text {
+    color: $text-color-1;
+    font-style: $font;
+    font-weight: 300;
+}
+
+// .nav {
+//     grid-area: nav;
+// }
+
+// .side-bar {
+//     grid-area: side-bar;
+// }
+
+// .main-content {
+//     grid-area: main-content;
+// }
+
+// .footer {
+//     grid-area: footer;
+// }
+
+
+
+.grid-container>div {
+
+    text-align: center;
+    padding: 20px 0;
+    font-size: 30px;
+}
+
+header {
+    // .header {
+        // display: grid;
+
+        .header-left {
+            // display: grid;
+            grid-area: header-left;
+            grid-column: 1 / 4;
+            grid-row: 1 / 1;
+
+            .header-left-1 {
+
+                h1 {
+                    @include header-text;
+                    animation: header 5s linear 2s infinite alternate; //The element to apply the animation to
+                    transition: transform 3000ms ease; // Transition 
+                }
+
+                h1:hover {
+                    transform: rotateY(360deg);
+                }
+            }
+
+            .header-left-2 {}
+        }
+
+        .header-right {
+            // display: grid;
+            grid-area: header-right;
+            grid-column: 4 / 6;
+            grid-row: 1 / 1;
+
+        }
+        
+    .nav-item {
+        grid-area: nav-bar;
```

Comment:
This isn't where you want to set this property.  There are multiple .nav-items in your list.  Instead set grid-area: nav-bar to the nav element (parent).